analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MemberOverrideCallsBaseMember.CSharp11.cs

```diff
@@ -0,0 +1,60 @@
+﻿
+record Derived : Base
+{
+    bool isInitialized;
+    int backingField;
+
```

Comment:
Can you explain what these testcases check?
More specifically, what makes this syntax different from C#9?

Reply:
It was a mistake on my part. I wanted to add test cases and realized that we cannot use override with static abstract and virtual in interfaces. Then I forgot to delete the file.